installation-guide/workstation.rst

```diff
@@ -24,11 +24,16 @@ A comfortable configuration would be:
 * Good video card (recent onboard Intel are ok, or dedicated NVidia/AMD)
 * Minimum requirement 2.14
 
-.. note:: 
+.. note::
   We recommend using the latest LTR version: 2.18.
```

Comment:
```suggestion
  We recommend using the latest LTR version: 3.4.
```

Reply:
I think we should adress this in another PR, to keep commits unitary.